tests/integrationv2/test_client_authentication.py

```diff
@@ -17,13 +17,8 @@
 @pytest.mark.parametrize("protocol", PROTOCOLS, ids=get_parameter_name)
 @pytest.mark.parametrize("certificate", ALL_TEST_CERTS, ids=get_parameter_name)
 def test_client_auth_with_s2n_server(managed_process, cipher, provider, curve, protocol, certificate):
-    host = "localhost"
     port = next(available_ports)
 
-    # NOTE: Currently do not support different signature schemes for client and server
-    if 'ecdsa' in certificate.cert:
-        pytest.skip("Skipping known failure, do not support different sig schemes for client and server")
-
```

Comment:
Was there a change previously made to fix this?

Reply:
The `invalid_test_parameters` filter detects this now. It will "uncollect" the test so we don't have to skip it manually.